addons/mail/static/src/core/web/chatter.js

```diff
@@ -194,11 +194,15 @@ export class Chatter extends Component {
                 } else {
                     this.state.showAttachmentLoading = false;
                     this.state.isAttachmentBoxOpened =
-                        this.props.isAttachmentBoxVisibleInitially && this.attachments.length > 0;
+                        this.state.isAttachmentBoxOpened && this.attachments.length > 0;
```

Comment:
I'm wondering if the fix shouldn't be line 178/181 also. Why do we have 2 different `useEffect` for the same thing?

The dependencies line 181 are messed up, `this.state.thread` is missing and `this.attachments` should be `this.attachments.length`.